.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    groups:
+      actions:
+        patterns:
+          - "*"
```

Comment:
Noticed this repo didn't have one of these but it's very convenient to have. It'll tend to run CIs once a month or every couple of months which is nice to have to check in. Not to mention actually having the actions stay up-to-date :)